### PR TITLE
Add proper license to all contracts that were unlicensed.

### PR DESCRIPTION
**Pull request by mijovic**

### CODE REVIEW

File: contracts/amm/VecMath.sol [hunk 1]
```diff
@@ -1,4 +1,4 @@
-// SPDX-License-Identifier: UNLICENSED
+// SPDX-License-Identifier: BUSL-1.1
```
Comment (mijovic) [inline contracts/amm/VecMath.sol:1]: Maybe?
```suggestion
// SPDX-License-Identifier: MIT
```

File: contracts/amm/unusedAMMImportOnly.sol [hunk 2]
```diff
@@ -1,4 +1,4 @@
-// SPDX-License-Identifier: UNLICENSED
+// SPDX-License-Identifier: BUSL-1.1
```
Comment (mijovic) [inline contracts/amm/unusedAMMImportOnly.sol:1]: This is not going to be deployed, no one should care about license here at all

File: contracts/debug/Debug.sol [hunk 3]
```diff
@@ -1,4 +1,4 @@
-// SPDX-License-Identifier: UNLICENSED
+// SPDX-License-Identifier: BUSL-1.1
```
Comment (mijovic) [inline contracts/debug/Debug.sol:1]: Maybe?
```suggestion
// SPDX-License-Identifier: MIT
```

File: contracts/math/Fixed256xVar.sol [hunk 4]
```diff
@@ -1,4 +1,4 @@
-// SPDX-License-Identifier: UNLICENSED
+// SPDX-License-Identifier: BUSL-1.1
```
Comment (mijovic) [inline contracts/math/Fixed256xVar.sol:1]:
```suggestion
// SPDX-License-Identifier: MIT
```

File: contracts/token/ERC20FixedSupply.sol [hunk 5]
```diff
@@ -1,4 +1,4 @@
-// SPDX-License-Identifier: UNLICENSED
+// SPDX-License-Identifier: BUSL-1.1
```
Comment (mijovic) [inline contracts/token/ERC20FixedSupply.sol:1]:
```suggestion
// SPDX-License-Identifier: MIT
```

File: contracts/token/ERC20OwnerMintableToken.sol [hunk 6]
```diff
@@ -1,4 +1,4 @@
-// SPDX-License-Identifier: UNLICENSED
+// SPDX-License-Identifier: BUSL-1.1
```
Comment (mijovic) [inline contracts/token/ERC20OwnerMintableToken.sol:1]:
```suggestion
// SPDX-License-Identifier: MIT
```

File: contracts/utils/UntrustedERC20.sol [hunk 7]
```diff
@@ -1,4 +1,4 @@
-// SPDX-License-Identifier: UNLICENSED
+// SPDX-License-Identifier: BUSL-1.1
```
Comment (mijovic) [inline contracts/utils/UntrustedERC20.sol:1]:
```suggestion
// SPDX-License-Identifier: MIT
```

File: contracts/debug/Debug.sol (hunk 3 above)
Comment (yuval-weiss) [reply contracts/debug/Debug.sol:1]: Yeah

**Comment by mijovic** [comment]
Rebased

**Pull request merged**